src/frontend/JasmineGraphFrontEnd.cpp

```diff
@@ -79,7 +94,17 @@ void *frontendservicesesion(std::string masterIP, int connFd, SQLiteDBInterface
         Utils utils;
         line = utils.trim_copy(line, " \f\n\r\t\v");
 
+        if (currentFESession > 1) {
+            canCalibrate = false;
+        } else if (currentFESession == 1) {
```

Comment:
Can we have only one else here?

Reply:
I think there is no need to specifically check for `currentFESession == 1` case. Because we anyway return true for all the cases that are `currentFESession =< 1`